modules/client-configuration/pages/registration-cli.adoc

```diff
@@ -18,11 +18,67 @@ Use the bootstrap script procedure to register traditional clients.
 For more information, see xref:registration-bootstrap.adoc[].
 ====
 
-This procedure requires that you have installed the [package]``salt-minion`` package on the Salt client before registration.
-For SLE 12 based clients, you also must have activated the [systemitem]``Advanced Systems Management`` module.
+This procedure requires that you have installed the [package]``venv-salt-minion`` (Salt bundle) or the [package]``salt-minion`` package on the Salt client before registration.
+Both use differently located configuration files, but the filenames are the same, and a differently named systemd service file.
```

Comment:
Both use configuration files in different locations and filenames remain the same.

Reply:
thanks, fixed locally.